src/components/apphost.js

```diff
@@ -235,77 +228,65 @@ const supportedFeatures = function () {
     const features = [];
 
     if (navigator.share) {
-        features.push('sharing');
+        features.push(AppFeature.Sharing);
     }
 
     if (!browser.edgeUwp && !browser.tv && !browser.xboxOne && !browser.ps4) {
-        features.push('filedownload');
+        features.push(AppFeature.FileDownload);
     }
 
     if (browser.operaTv || browser.tizen || browser.orsay || browser.web0s) {
-        features.push('exit');
-    } else {
-        features.push('plugins');
```

Comment:
Was the elimination of the plugins feature intended?

Reply:
Yes I couldn't find any usages of it. 🤷‍♂️